src/js/components/Select/SelectContainer.js

```diff
@@ -342,9 +352,37 @@ class SelectContainer extends Component {
     return result;
   };
 
+  renderClear = () => {
+    const { clear, theme } = this.props;
+    const { renderValue, label } = clear;
+    if (renderValue) {
+      return renderValue({ onClear: this.clearSelection });
+    }
+    return (
+      <Box flex={false}>
+        <Button
+          tabIndex="-1"
+          hoverIndicator="background"
+          onClick={this.clearSelection}
+        >
+          <Box
+            background="light-4"
+            {...theme.select.options.box}
+            {...theme.select.options.container}
+          >
+            <Text {...theme.select.options.text}>
+              {label || 'Clear selection'}
```

Comment:
Is it possible to have 'Clear selection' as the default `label`, and it could be overridden by the user.
```suggestion
              {label}
```

Reply:
not sure how since the label is either a boolean or an object.

Reply:
Then we might need to edit doc.js where it says that `label` is a string.
```        
label: PropTypes.string
          .description('Label for the clear selection item')
          .defaultValue('Clear selection'),
```

Reply:
damn sorry i meant that clear is either a boolean or an object.

Reply:
doc is updated, but as @oorestisime said, can't default it higher up in the component structure because it could be boolean or object.